### PR TITLE
Issue #30 - Expose Deserializer to API

### PR DESCRIPTION
**Pull request by frisk**

### CODE REVIEW

**Comment by SeyZ** [comment]
Hey @frisk thank you for the PR!

**Comment by SeyZ** [comment]
@frisk Can you just add a CHANGELOG line in the file https://github.com/ForestAdmin/forest-express/blob/devel/CHANGELOG.md before merging 👍 ?

**Comment by SeyZ** [comment]
@frisk Thank you! You rock!

**Comment by frisk** [comment]
@SeyZ It's a small one, but it will help me out a lot with some of the backend customization I need to do! I've updated the CHANGELOG as you requested. I've also put up another PR dependent upon this one in https://github.com/ForestAdmin/forest-express-sequelize/pull/45.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Deserializer - Expose Deserializer module to API
+
+## RELEASE 0.1.31 - 2016-11-17
 ### Fixed
 - Custom Actions - Fix missing actions for Smart Collections.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: This fix has not been released yet, I'll fix the Changelog in the master branch.

**Pull request merged**